src/util/motionModel/packaging.ts

```diff
@@ -175,6 +177,7 @@ export const packMetadata = async (
         keyframeDistance: getConfig().DX,
         resolution: '2k',
         version: '1.8',
+        deviceAnonymousId: deviceAnonymousId,
```

Comment:
we can also name it smth simple like `deviceId`,
since it will be a name to go full stack,
but let me know if you were asked to make it like this